docs/DeveloperGuide.md

```diff
@@ -188,7 +188,11 @@ Predicate<Person> composedPredicate = predicate1.and(predicate2);
 
 ## **Documentation, logging, testing, configuration, dev-ops**
 
-Coming soon.
+* [Documentation guide](Documentation.md)
+* [Testing guide](Testing.md)
```

Comment:
In testing.md, perhaps we can update the package names e.g. seedu.address.commons.StringUtilTest